en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md

```diff
@@ -0,0 +1,286 @@
+               <property expression="json-eval($.containerName)" name="containerName" scope="default" type="STRING"/>
+               <msazurestorage.init>
+                   <accountName>eiconnectortest</accountName>
+                   <accountKey>bWt69gFpheoD6lwVsMgeV5io2/KxlXK1KUcod68PhzuV1xHxje0LBD4Bd+y+ESAOlH5BTAfvdDG5q4Hhg==</accountKey>
+               </msazurestorage.init>
+               <msazurestorage.deleteContainer>
+                   <containerName>{$ctx:containerName}</containerName>
+               </msazurestorage.deleteContainer>
+               <log level="full">
+                   <property name="Delete selected container" value="Delete selected container"/>
+               </log>
+               <respond/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+   </api>
+   ```
+**Note**
+
+* As `accountKey` use the access key obtained from setting up the Microsoft Azure Storage account.
+* As `accountName` get the name of created **Storage Account** inside the Microsoft Azure Storage account.
+    
+Now we can export the imported connector and the API into a single CAR application. CAR application is the one we are going to deploy to server runtime. 
+   
+{!/references/connectors/exporting-artifacts.md!}
+
+## Get the project
+
+You can download the ZIP file and extract the contents to get the project code.
+
+<a href="../../../../assets/attach/connectors/ms-azure-connector.zip">
+    <img src="../../../../assets/img/connectors/download-zip.png" width="200" alt="Download ZIP">
+</a>
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+
+## Testing
+
+Invoke the API as shown below using the curl command. Curl Application can be downloaded from [here] (https://curl.haxx.se/download.html).
+
+1. Creating a new container in Microsoft Azure Storage for store employee details.
+ 
+   **Sample request**
+
+   `curl -v POST -d {"containerName":"employeedetails"} "http://localhost:8290/resources/createcontainer" -H "Content-Type:application/json"`
+
+   **Expected Response**
+   `{
+       "success": true
+    }`
+    
+2. Retrieve information about the created containers.    
+   
+   **Sample request**
+   
+   `curl -v POST -d {} "http://localhost:8290/resources/listcontainer" -H "Content-Type:application/json"`
+   
+   **Expected Response**
+   
+     It will retrieve all the existing container names.
+     
+     `{
+         "result":{
+            "container":[
+               "employeedetails",
+               "employeefinancedetails"
+            ]
+         }
+      }`
+
+3. Upload text or binary employee data (blob).
+
+   **Sample request**
+   
+     `curl -v POST -d {"containerName": "employeedetails","fileName": "sample.txt","filePath": "/home/kasun/Documents/MSAZURESTORAGE/sample.txt"} "http://localhost:8290/resources/adddetails" -H "Content-Type:application/json"`
```

Comment:
Let's mention `/home/kasun/Documents/MSAZURESTORAGE/sample.txt` should be a valid path to a text file containing employee information.

Reply:
Updated.